src/plugins/controls/public/options_list/components/options_list_popover_sorting_button.tsx

```diff
@@ -33,10 +33,52 @@ import { useOptionsList } from '../embeddable/options_list_embeddable';
 interface OptionsListSortingPopoverProps {
   showOnlySelected: boolean;
 }
+
 type SortByItem = EuiSelectableOption & {
   data: { sortBy: OptionsListSortBy };
 };
 
+const sortOrderOptions: EuiButtonGroupOptionProps[] = [
+  {
+    id: 'asc',
+    iconType: `sortAscending`,
+    'data-test-subj': `optionsList__sortOrder_asc`,
+    label: OptionsListStrings.editorAndPopover.sortOrder.asc.getSortOrderLabel(),
+  },
+  {
+    id: 'desc',
+    iconType: `sortDescending`,
+    'data-test-subj': `optionsList__sortOrder_desc`,
+    label: OptionsListStrings.editorAndPopover.sortOrder.desc.getSortOrderLabel(),
+  },
+];
+
+const SortButton = ({
```

Comment:
Similar to the bug with `SwitchWithTooltip` mentioned in the PR description, the `SortButton` component was being re-rendered **way more often** than it needed to be - so, I moved it out to be its own component to limit this as much as possible.